packages/picker-button/README.md

```diff
@@ -0,0 +1,157 @@
+## Description
+
+An `<sp-picker-button>` is used as a sub-component of patterns like the `<sp-combobox>` (release pending) to pair a button interface with a text input. With its many custom states and alterations, it isn't likely to be leveraged directly outside of more complex UIs.
+
+### Usage
+
+[![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/picker-button?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/picker-button)
+[![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/picker-button?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/picker-button)
+
+```
+yarn add @spectrum-web-components/picker-button
+```
+
+Import the side effectful registration of `<sp-picker-button>` via:
+
+```
+import '@spectrum-web-components/picker-button/sp-picker-button.js';
+```
+
+When looking to leverage the `PickerButton` base class as a type and/or for extension purposes, do so via:
+
+```
+import { PickerButton } from '@spectrum-web-components/picker-button';
+```
+
+## Sizes
+
+<sp-tabs selected="m" auto label="Size Attribute Options">
+<sp-tab value="s">Small</sp-tab>
+<sp-tab-panel value="s">
+
+```html
+<sp-picker-button size="s"></sp-picker-button>
+<br />
+<sp-picker-button size="s"><span slot="label">All</span></sp-picker-button>
+```
+
+</sp-tab-panel>
+<sp-tab value="m">Medium</sp-tab>
+<sp-tab-panel value="m">
+
+```html
+<sp-picker-button size="m"></sp-picker-button>
+<br />
+<sp-picker-button size="m"><span slot="label">All</span></sp-picker-button>
+```
+
+</sp-tab-panel>
+<sp-tab value="l">Large</sp-tab>
+<sp-tab-panel value="l">
+
+```html
+<sp-picker-button size="l"></sp-picker-button>
+<br />
+<sp-picker-button size="l"><span slot="label">All</span></sp-picker-button>
+```
+
+</sp-tab-panel>
+<sp-tab value="xl">Extra Large</sp-tab>
+<sp-tab-panel value="xl">
+
+```html
+<sp-picker-button size="xl"></sp-picker-button>
+<br />
+<sp-picker-button size="xl"><span slot="label">All</span></sp-picker-button>
+```
+
+</sp-tab-panel>
+</sp-tabs>
+
+## Text and icon
+
+With the use of the `label` slot, you can deliver an `<sp-picker-button>` with both an icon and a text label:
+
+```html
+<sp-picker-button><span slot="label">All</span></sp-picker-button>
+```
+
+## Icon only
+
+Without content addressed to the `label` slot, the `<sp-picker-button>` with both an icon and a text label:
+
+```html
+<sp-picker-button></sp-picker-button>
+```
+
+### Custom icon
+
+You can customize the icon in an `<sp-picker-button>` by addressing a new icon to the `icon` slot:
+
+```html
+<sp-picker-button><sp-icon-add slot="icon"></sp-icon-add></sp-picker-button>
+```
+
+## Modifying attributes
+
+### Rounded
+
+When delivered as part of the `express` theme, an `<sp-picker-button>` with the `rounded` attribute will be given deeply rounded corners:
```

Comment:
This UI doesn't seem to be any different from Spectrum classic? I compared in storybook. Is that a CSS token issue?

Reply:
Classic:
<img width="1000" alt="image" src="https://user-images.githubusercontent.com/1156657/194341503-7177cb56-28de-470a-b924-ebcedf5f6b48.png">

Express:
<img width="989" alt="image" src="https://user-images.githubusercontent.com/1156657/194341642-457b2e5b-b729-430b-8c86-10f7fa0f9643.png">